tests/terraform/manual_upgrade/upgradecluster_test.go

```diff
@@ -0,0 +1,396 @@
+package upgradecluster
```

Comment:
We can leave this as upgradecluster and ii would be nice if you could rename the existing upgradecluster test to sucupgrade. Please change directory names accordingly.

Reply:
+1